nestkernel/simulation_manager.cpp

```diff
@@ -205,7 +205,7 @@ nest::SimulationManager::set_status( const DictionaryDatum& d )
   {
     if ( nprelim < 0 )
       LOG( M_ERROR,
-        "Scheduler::set_status",
+        "Network::set_status",
```

Comment:
Thanks for the catch! Also not quite correct: should be `"SimulationManager::set_status"`